functests/4_latency/latency.go

```diff
@@ -413,3 +425,21 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 
 	return latencies[1]
 }
+
+func printLogFile(node *corev1.Node, logName string) string {
```

Comment:
nit: this function doesn't `print`. It `get`s.

Reply:
@fromanirh @MarSik @cynepco3hahue  
TL;DR - we need to call this function in case of any error.
The current implementation is getting the log only in case the expected latency > actual.
But the case might fail for various other reasons.
For example, in hwlatdetect it would compare the value (inside the tool) and exit with an error if the measured value is bigger than the expected, so the log will never get printed.

Reply:
works for me

Reply:
I agree with Talor, I'll handle that.